lint/clang_tidy.bzl

```diff
@@ -349,6 +391,7 @@ def lint_clang_tidy_aspect(binary, configs = [], global_config = [], header_filt
             passes these as -isystem. Change this to False to pass these as -I, which allows clang-tidy to regard
             them as regular header files.
         verbose: print debug messages including clang-tidy command lines being invoked.
+        mnemonic_suffix: suffix of mneomnic to be used. A prefix of AspectRulesLint is always used.
```

Comment:
could you make a separate PR for this change pls? lumping multiple independent things into one PR makes it harder to review and less clear what's related.
In that PR I'll probably ask why clang_tidy is the only aspect with a customizable mnemonic.

Reply:
That's a good suggestion, have done so